models/src/main/java/com/connexta/transformation/rest/models/ErrorMessage.java

```diff
@@ -25,6 +27,7 @@
 
 /** Error message. */
 @ApiModel(description = "Error message.")
+@JsonInclude(Include.NON_NULL)
```

Comment:
In the Last 3 pushes I made I added this line to ErrorMessage, ErrorResponse, MetadataInformation, and TransformationPollResponse so that null fields won't be printed in responses. I also changed the transform 201 description to say "created" instead of "accepted".